exporter/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM zenato/puppeteer
```

Comment:
Can we have a more specific version instead of latest?

Reply:
I see there are plenty of puppeteer images. See https://hub.docker.com/search/?isAutomated=0&isOfficial=0&page=1&pullCount=0&q=puppeteer&starCount=0. 
The most popular one has tags.

Reply:
I first went with the most popular one `alekzonder/puppeteer` but it uses an outdated version of puppeteer. Hence, my choice. But the 2nd one could work because it uses the latest puppeteer - https://hub.docker.com/r/buildkite/puppeteer/. Will try now